DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -562,15 +558,11 @@ public function getXsdValidationBasePath()
     /**
      * Loads a cache driver.
      *
-     * @param string $cacheName         The cache driver name
-     * @param string $objectManagerName The object manager name
-     * @param array  $cacheDriver       The cache driver mapping
-     *
      * @throws InvalidArgumentException
      *
      * @psalm-suppress UndefinedClass this won't be necessary when removing metadata cache configuration
      */
-    protected function loadCacheDriver($cacheName, $objectManagerName, array $cacheDriver, ContainerBuilder $container): string
+    protected function loadCacheDriver(string $cacheName, string $objectManagerName, array $cacheDriver, ContainerBuilder $container): string
```

Comment:
[`AbstractDoctrineExtension::loadCacheDriver()` already has `string` in 5.4](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php#L349)